search-commons/src/test/java/no/unit/nva/search/IndexDocumentTest.java

```diff
@@ -97,33 +47,16 @@ private Publication createSampleBookInABookSeriesFromAPublisher(PublishingHouse
         return createPublicationWithEntityDescription(entityDescription);
     }
 
-    @Test
-    public void fromPublicationCanConvertReferenceFromPublicationWithoutLoss() throws Exception {
-
-        Publication publication = createSampleBookInABookSeriesFromAPublisher(publishingHouseWithUri());
-
-        IndexDocument actualDocument = IndexDocument.fromPublication(publication);
-        assertThat(actualDocument, doesNotHaveEmptyValuesIgnoringFields(IGNORED_INDEXED_DOCUMENT_FIELDS));
-        assertEquals(actualDocument.getReference(), publication.getEntityDescription().getReference());
-    }
 
     @Test
-    public void toJsonPreservesRegistredDataDataWhenPublisherIsUnconfirmed() throws Exception {
-
-        Publication publication = createSampleBookInABookSeriesFromAPublisher(unconfirmedPublishingHouse());
-
+    public void fromPublicationAndBack() throws Exception {
+        Publication publication = createSampleBookInABookSeriesFromAPublisher(publishingHouseWithUri());
```

Comment:
This test can pass any time if the created publication is an empty object. Is it possible to ensure that all fields all filled?